### ISSUE TITLE
Metrics on how chrome://tracing is used.

### ISSUE THREAD

**Issue by tdresser** [catapult-project/catapult#2121]
Do we have any way of gathering metrics on how chrome://tracing is used?

@natduca and I were discussing removing the input latency tab, which I thought would be fine, but today I just saw someone using it. It would be great if I could easily measure how frequently that feature is used.

**Comment by nedn** [comment]
@paulirish how does devtool team go about  "can we remove feature X"  in general?

**Comment by anniesullie** [comment]
Would it be okay to use Google analytics for collecting metrics?

@rschoen any ideas on if there's a policy here?

**Comment by paulirish** [comment]
@nedn in devtools, we have a few UMA histograms in place, but not nearly enough that would help indicate usage for feature deprecations. We haven't explored google analytics for real, though personally I'd really enjoy having those numbers available.

**Comment by natduca** [comment]
I could imagine us leaning on uma. But tbh, I think in the case of this, we
should just err on the side of doing our best to make the call, and undoing
and/or learning from any mistakes we make... the real fix here is us
becoming not part of chrome:// urls, I think?

**Comment by anniesullie** [comment]
@natduca I don't understand how us becoming not part of chrome://urls makes a difference here?

If we used Google analytics, we should be able to get data on usage regardless of whether users are poking at vulcanized traces, chrome://tracing, some future extension-based trace-viewer, etc.

**Comment by natduca** [comment]
When we're a chrome url, we don't have access to real internet, iirw. You can basically only talk to chrome via chrome.send.

**Comment by anniesullie** [comment]
Thanks, I didn't realize that.

**Comment by tdresser** [comment]
In the particular case I brought up, I agree, we can just do something that seems sensible.
Eventually I think it would be nice to have some kind of feedback loop here though.

**Comment by natduca** [comment]
I agree. However, I think I'm going to WontFix because this is quite low on our list compared to all the things. :)

**Comment by benshayden** [comment]
This is still low priority, but we now have questions about how trace viewer is being used that analytics/UMA would help answer.
- Is anybody using tquery?
- Is anybody using ns time display mode?

Next steps:
1. Add tr.b.Timing instrumentation around tquery and time display mode
1. Add Timing instrumentation around other UI features to measure usage
1. Add Timing marks around drawing code and other performance-critical sections
1. Use chrome.send to plumb Timing marks and instant events to UMA
